.circleci/config.yml

```diff
@@ -41,55 +46,12 @@ jobs:
       - store_test_results:
           path: reports/junit
 
-  test-node-12:
-    working_directory: ~/jest
-    executor: node/default
-    steps:
-      - checkout
-      - node/install:
-          node-version: '12'
-          install-npm: false
-      - node/install-packages: *install
-      - run:
-          command: yarn test-ci-partial
-      - store_test_results:
-          path: reports/junit
-
-  test-node-14:
-    working_directory: ~/jest
-    executor: node/default
-    steps:
-      - checkout
-      - node/install:
-          node-version: '14'
-          install-npm: false
-      - node/install-packages: *install
-      - run:
-          command: yarn test-ci
```

Comment:
this has a different command (collects coverage) - can we get that in (or just keep v14 separate)?

Reply:
Oops, I missed that. Good point, I'll create two jobs and make partial runs explicit.

Reply:
Done! We now have two matrices:

- test-node-x
- test-node-partial-x